### ISSUE TITLE
feedback on ch 2

### ISSUE THREAD

**Issue by murphyk** [Ma-Lab-Berkeley/deep-representation-learning-book#4]
p36 Fig 2.1 Maybe put this in ch1?

p38. It is odd that the GT parameters are denoted by U and the estimated
parameters are denoted by U*. Usually the estimated parameters are
denoted Uhat, and U* is for the GT.
(I guess you use * to denote the optimal estimate,
whereas hat is a generic (possibly suboptimal) estimate?
Maybe write GT as U_{gt}?)

p39. "Model selection is a difficult problem and, nowadays in the
era of deep learning where it is called
hyperparameter optimization, is usually done via brute
force or Bayesian optimization."
That does not address the more fundamental issue of what loss
function to optimize when doing model selection.
Maybe mention reconstruction error on a  validation set
as a simple method?

p41. Cite PPCA papers.

p42. Eq 2.1.36. This is a very cute example :)

p43. Term "eigenface" not used before.

p46. Here you use Bern(theta) but in eqn 1.3.13 on p15 you use B(1,p)

p48. It is not clear what is the connection between V and U.

p52. Maybe draw a diagram of soft-thresholding function in 1d,
to show how it snaps small signals to 0, and visually compare to relu.

p55. Eqn 2.3.24. The use of f for a generic encoder
here clashes with the LISTA definition of f.
Maybe use e instead of f?

p56. "This algorithm is the simplest case in which a clear divide between forward
optimization and backward learning manifests". Maybe explain that A is a fixed
dimensional  object that is shared across data, whereas Z grows / changes
with the dataset, so it is natural to treat them asymmetrically.
